torch/nn/init.py

```diff
@@ -384,6 +384,9 @@ def kaiming_uniform_(tensor, a=0, mode='fan_in', nonlinearity='leaky_relu'):
         >>> w = torch.empty(3, 5)
         >>> nn.init.kaiming_uniform_(w, mode='fan_in', nonlinearity='relu')
     """
+    if 0 not in tensor.shape:
+        warnings.warn("Initializing zero-dimensional tensors is a no-op")
```

Comment:
I think this should be "zero-element" tensors instead of "zero-dimensional".

Reply:
Agreed, fixed in https://github.com/pytorch/pytorch/pull/56505/commits/9f86953b32fea04694eccf2b0d711820d05a0fd8